modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -996,6 +1007,26 @@ private void _validateName(
 		}
 	}
 
+	private void _validateRelationship(long objectFieldId, boolean required)
+		throws ObjectFieldRelationshipTypeException {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipPersistence.fetchByObjectFieldId2(objectFieldId);
+
+		if ((objectRelationship != null) &&
+			StringUtil.equals(
+				objectRelationship.getDeletionType(),
+				ObjectRelationshipConstants.DELETION_TYPE_DISASSOCIATE) &&
+			StringUtil.equals(
+				objectRelationship.getType(),
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY) &&
+			required) {
+
+			throw new ObjectFieldRelationshipTypeException(
+				"Object field cannot be mandatory");
```

Comment:
Can you change this message to:
Object field cannot be required because the relationship deletion type is disassociate.